docs/index.md

```diff
@@ -0,0 +1,253 @@
+    of a zTOC. See [zTOC](#ztoc) section.
+
+    **Note: the ordering MUST be consistent with the ordering of the layers from the OCI image. This consistent ordering ensures index building is deterministic.**
+
+* `annotations` _string-string map_
+
+    An OPTIONAL OCI image manifest property which contains arbitrary metadata for the SOCI index.
+
+    * `"com.amazon.soci.build-tool-identifier"` can be used to identify the version of SOCI CLI used to generate the index.
+
+### Entities and Relationships
+
+The following figure shows the relationship between an OCI image and its SOCI index:
+
+![entities-and-relations](images/soci-index-entities-and-relationships.drawio.svg)
+
+### Example Serialized SOCI Index
+
+_Default SOCI Index for
+public.ecr.aws/docker/library/rabbitmq@sha256:8af9cc0bd40bcd466e5935e25c08ad2306f9a25091b0d145492d0696888851dc:_
+
+```
+{
+    "annotations": {
+        "com.amazon.soci.build-tool-identifier": "AWS SOCI CLI v0.1"
+    },
+    "config": {
+        "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+        "mediaType": "application/vnd.amazon.soci.index.v1+json",
+        "size": 2
+    },
+    "layers": [
+        {
+            "annotations": {
+                "com.amazon.soci.image-layer-digest": "sha256:aece8493d3972efa43bfd4ee3cdba659c0f787f8f59c82fb3e48c87cbb22a12e",
+                "com.amazon.soci.image-layer-mediaType": "application/vnd.oci.image.layer.v1.tar+gzip"
+            },
+            "digest": "sha256:cb011209b93fde5a7a698302ca2e2dd4928278da32a31995c5df6d16cb27c638",
+            "mediaType": "application/octet-stream",
+            "size": 1229704
+        },
+        {
+            "annotations": {
+                "com.amazon.soci.image-layer-digest": "sha256:e8b79c92d68de7b64f81149d112547255d8edecba2f64054a533954ee3c106fe",
+                "com.amazon.soci.image-layer-mediaType": "application/vnd.oci.image.layer.v1.tar+gzip"
+            },
+            "digest": "sha256:7ae9f4713a78a4895a7a522117c552a519d8289eecc446820b1b528cf9257b82",
+            "mediaType": "application/octet-stream",
+            "size": 841192
+        },
+        {
+            "annotations": {
+                "com.amazon.soci.image-layer-digest": "sha256:bb157f88c818b56b154696493271c0d7a2704ed253fc2111d0a96fa89bf44984",
+                "com.amazon.soci.image-layer-mediaType": "application/vnd.oci.image.layer.v1.tar+gzip"
+            },
+            "digest": "sha256:715ee958c9cff9bca859e940cb05a3937bd2a6d2570fec910602df095339b067",
+            "mediaType": "application/octet-stream",
+            "size": 1086672
+        }
+    ],
+    "mediaType": "application/vnd.oci.image.manifest.v1+json",
+    "schemaVersion": 2,
+    "subject": {
+        "digest": "sha256:ab5a62720aa71c964aae400825284ca3e8229934928394e68a06e395a225faae",
+        "mediaType": "application/vnd.oci.image.manifest.v1+json",
+        "size": 2752
+    }
+}
+```
+
+### Example Serialized zTOC
```

Comment:
nit: I think we should have a note that mentions that this a _human readable_ format the resembles a `zTOC`, with some parts like the compression checkpoint information redacted. The actual `zTOC` is serialized as a binary file.

Reply:
Added in latest revision.